love/src/components/MainTel/CameraHexapod/CameraHexapod.container.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import CameraHexapod from './CameraHexapod';
+import SubscriptionTableContainer from 'components/GeneralPurpose/SubscriptionTable/SubscriptionTable.container';
+import { addGroup, removeGroup } from 'redux/actions/ws';
+import { getHexapodStatus, getHexapodTables } from 'redux/selectors';
+
+export const schema = {
+  description: 'View of Camera Hexapod',
+  defaultSize: [40, 23],
+  props: {
+    title: {
+      type: 'string',
+      description: 'Name displayed in the title bar (if visible)',
+      isPrivate: false,
+      default: 'Camera Hexapod',
+    },
+    salindex: {
+      type: 'number',
+      description: 'Salindex of the Hexapod',
```

Comment:
I would add info about which index to use:
"Salindex of MTHexapod. 1 for Camera Hexapod and 2 for M2 Hexapod"